data/human/human missions.txt

```diff
@@ -6301,3 +6301,410 @@ mission "Blind Man from Martini"
+			`	Before going more than three steps, she stops and turns back to you. "Oh, sorry - I'll authorize your payment as soon as I get back to my room. I don't have it with me."`
+			`	Sure enough, less than five minutes after she disappears into a building in the distance, your communicator bleeps and reports that <payment> has been credited to your account.`
+			action
+				set "delivered torrey's package to irene"
+
+mission "Take Me To The South A: Followup A"
+	landing
+	invisible
+	source "Greenrock"
+	to offer
+		has "FW Pirates: Attack 2: done"
+		has "delivered torrey's package to irene"
+	on offer
+		conversation
+			`As Tomek heads toward Alondo's ship, you notice some other ships landing nearby - several of which look in much worse shape. Between them is a bustle of activity that looks like a kind of controlled panic, setting up a large tent and bringing various supplies and tables into it. After a few minutes, it becomes clear that they are setting up a makeshift triage center to get the worst wounded to the ships with the best medical facilities as quickly as possible.`
+			`	Almost against your will, you find your steps drawn in that direction. Staying carefully out of the way of the hurrying corpsmen and nurses, you watch as the triage tent begins to fill up with the walking wounded, the not-so-walking wounded...and a few gurneys that someone pulls a sheet over not long after they arrive. Before long, you spot someone you think you recognize: Irene Brower, the gunnery sergeant you delivered a care package to before the war.`
+			`	She is lying on a stretcher placed on what appears to be a folding cafeteria table, with an IV in one arm and a makeshift bandage over the other shoulder, already soaked through with blood. The lower half of her body is covered by a bloody sheet...and by the shape the sheet makes, you can tell immediately that there's something very wrong. Her eyelids flicker for a moment, then she opens her eyes and focuses on you. "Hey...I know you. You brought me that package from Tor...thanks. Meant a lot." She attempts to raise her head, then thinks better of it, though her eyes still flick downward. "Might be...the last of home I get to see." Her eyes close again, and a pair of nurses comes over and shoos you out of the way so they can start moving Irene. As they carry her off, careful to keep her IV bag held high, you hear her start coughing, and they head to what appears to be the largest of the ships nearby.`
+				decline
+
+mission "Take Me To The South A: Followup A1"
+	landing
+	invisible
+	to offer
+		has "Take Me To The South A: Followup A: declined"
+		has "FW Pirates: Attack 3: done"
+	on offer
+		conversation
+			`Now that you have a minute to breathe, you turn to your computer and pull up the casualty lists from the Battle of Greenrock. You only have to scroll through it for a few seconds before you see what you are looking for and dreading:`
```

Comment:
I really like the fact that the player gets to know people who fought and died for the FW's effort. It reminds me of the Beelzebub missions.